src/core_plugins/kibana/common/field_formats/types/_numeral.js

```diff
@@ -1,6 +1,6 @@
 import _ from 'lodash';
-import numeral from 'numeral';
-import { FieldFormat } from 'ui/index_patterns/_field_format/field_format';
+import numeral from '@spalger/numeral';
```

Comment:
Can this wait for a separate PR? This seems like an existing artifact that is outside the scope of this PR and I am trying to get this PR finished ASAP since another feature is dependent on it and Tuesday is feature cutoff.